### PR TITLE
Added Javadoc to all methods

### PR DESCRIPTION
**Pull request by broken1arrow**
Added javadoc to briefly explain all methods works and arguments. 

No code changes.

### CODE REVIEW

File: src/main/java/com/tchristofferson/configupdater/ConfigUpdater.java [hunk 1]
```diff
@@ -261,6 +347,15 @@ private static String addIndentation(String s, String indents) {
         return builder.toString();
     }
 
+    /**
+     * Writes the specified commit to the provided buffer writer. If the kommit exist for this path.
```
Comment (tchristofferson) [inline src/main/java/com/tchristofferson/configupdater/ConfigUpdater.java:351]: typo in kommit, should be commit

**Comment by broken1arrow** [comment]
Alright solve that small typo.

**Comment by tchristofferson** [comment]
@broken1arrow If you request anymore pull requests make it for the develop branch.

**Comment by broken1arrow** [comment]
> @broken1arrow If you request anymore pull requests make it for the develop branch.

Alright, will do it next time.

**Pull request merged**